src/CrowdinApiClient/CollectionIterator.php

```diff
@@ -40,13 +41,14 @@ public function current()
     /**
      * Move forward to next element
      * @link https://php.net/manual/en/iterator.next.php
-     * @return mixed Next item
+     * @return void Any returned value is ignored.
      * @since 5.0.0
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
-        $var = next($this->_items);
-        return $var;
```

Comment:
probably this can break some existing integrations

Reply:
Hmm, okay. I can revert that. I was basing it off of the docs which return void. 

Ref: https://www.php.net/manual/en/iterator.next.php

Reply:
I'm just thinking that some users may rely on the returned result, isn't it?

Reply:
@mlahargou please leave a TODO there just to not forget to remove this on the major version release

Reply:
I guess if they are digging into the `CollectionIterator` itself. Really that should have the exact return type that the `Iterator` interface has. We can just suppress the error for now, but that will have to fixed eventually I believe.

I reverted my original change there, so all should be the same behavior.

Reply:
> please leave a TODO there just to not forget to remove this on the major version release

Totally! I can add that to all of the `#[\ReturnTypeWillChange]` locations. Just double checking, the format is `@TODO Remove the 'ReturnTypeWillChange' suppression on next major version`, right?

Reply:
@mlahargou yes, the suggested TODO format looks great

Reply:
Done!